### ISSUE TITLE
[Problem] Techdraw: 'Print center marks' function does not work

### ISSUE THREAD

**Issue by DEL-TRONIK** [FreeCAD/FreeCAD#11309]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

From the Techdraw workbench, when exporting a drawing, although the 'Print center marks' function is deactivated in the Techdraw preferences (Annotation tab), they are always displayed when exporting a drawing in PDF format.

### Full version info

```shell
[code]
OS: Debian GNU/Linux 12 (bookworm) (GNOME/gnome-xorg)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.34873 (Git) AppImage
Build type: Release
Branch: main
Hash: 889c8c49f1bbc0910c8dc32a512f4c6fbedeb8b9
Python 3.10.13, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: English/United States (en_US)
Installed mods: 
  * kicadStepUpMod 10.19.5
  * Defeaturing 1.2.0
  * ThreadProfile 1.89.0
  * fasteners 0.4.66
  * ExplodedAssembly
[/code]
```

### Subproject(s) affected?

Techdraw

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Duplicate of #11228 ?

**Comment by DEL-TRONIK** [comment]
I don't think so, it's not a detailed view. The problem is that the 'Print center marks' function in the preferences does not work (even if it's unchecked it still displays the center marks, which should not be the case). I just want to display the center marks when working in Techdraw, but not when exporting the drawing.

**Comment by WandererFan** [comment]
[](https://forum.freecad.org/viewtopic.php?t=82532)

**Comment by WandererFan** [comment]
FYI, you can use https://wiki.freecad.org/TechDraw_CosmeticCircle instead.  It seems to put the circle in the right place.

**Comment by WandererFan** [comment]
Should be fixed by PR #11325
